internal/pikamon/commands/root.go

```diff
@@ -30,16 +28,16 @@ func Handle(s *discordgo.Session, m *discordgo.MessageCreate) {
 	text := strings.TrimSpace(strings.ToLower(m.Content))
 
 	// ignore all messages not prefixed with bot command keyword
-	if !strings.HasPrefix(text, commandKeyword) {
+	if !strings.HasPrefix(text, util.CommandKeyword) {
 		return
 	}
-	commandText := strings.TrimSpace(text[len(commandKeyword):])
+	commandText := strings.TrimSpace(text[len(util.CommandKeyword):])
 	if commandText == "" {
 		return
 	}
 
 	// call the appropriate handler based on the root command
-	command := strings.SplitN(commandText, " ", 1)[0]
+	command := strings.Fields(commandText)[0]
```

Comment:
I believe this will cause a panic if no command is specified, and just the command keyword is passed.

Reply:
Put in a check for this scenario